edb/lib/ext/auth.edgeql

```diff
@@ -30,6 +30,11 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
         create constraint exclusive on ((.iss, .sub))
     };
 
+    create type ext::auth::LocalIdentity {
+        create property password_hash: std::str;
+        create required link identity: ext::auth::Identity;
```

Comment:
Do we want to make this link unique 1:1 instead of N:1? During authentication, I think we will first select the identity (corresponding to the user choosing to log in with a password), then check if the password matches. It would be weird if the schema allows multiple passwords per identity.

Reply:
Yeah, that makes sense, nice catch. `create constraint exclusive` right?